github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-postsubmits.yaml

```diff
@@ -11,7 +11,7 @@ postsubmits:
       extra_refs:
       - org: kubevirt
         repo: project-infra
-        base_ref: main
+        base_ref: master 
```

Comment:
This is because project infra hasn't moved to `main` yet right?

Reply:
Yep